anet-dictionary.yml

```diff
@@ -640,7 +662,7 @@ dictionary:
 
       position:
         type: ANET Administrator
-
+  liftedCustomFieldPrefix: cF_
```

Comment:
Wouldn't it be easier to just specify a list of 'lifted custom fields', e.g.:
```
      liftedCustomFields:
      - multipleButtons
      - inputFieldName
      - …etc…
```
?

Reply:
We want to order normal fields and custom fields mixed up. That is why i need them in the same block

Reply:
Then why not combine the two lists (`fieldsOnShowPage` and `liftedCustomFields`) into one list to specify which fields (standard & custom) need to be shown in which order on the Show page?
Just having this extra list prop makes field definitions the same again for all object types, and adding similar functionality to other Show pages would involve just adding such a list to an object type (and adapting its Show page to use it, obviously). And then it makes sense to rename `numberOfFieldsUnderAvatar` to something like `numberOfFieldsIn FirstColumn`.

Reply:
Ah yes we are getting to the critical part. I think i wanted to separate the custom and non-custom ones, because there was already a customField parser component. `ReadonlyCustomFields`. Because i had the info of non-custom component fields and props in show page, I can map each non-custom component like this:
```
          return Object.keys(Person.shownNonCustomFields).reduce(
            (accum, key) => {
              accum[key] = (
                <Field
                  name={key}
                  label={Person.shownNonCustomFields[key].label}
                  component={FieldHelper.ReadonlyField}
                  humanValue={humanValuesExceptions[key]}
                  className={classNameExceptions[key]}
                />
              )

              return accum
            },
            {}
          ) 
```
I can't map custom fields like this, because there are configs to go through, and i wanted to reuse the already built-in config parsing file `CustomFields.js`